services/sysinfo/client/client.go

```diff
@@ -131,6 +132,7 @@ func (p *dmesgCmd) SetFlags(f *flag.FlagSet) {
 	f.Int64Var(&p.tail, "tail", -1, "tail the latest n lines")
 	f.BoolVar(&p.ignoreCase, "i", false, "ignore case")
 	f.BoolVar(&p.invertMatch, "v", false, "invert match")
+	f.Int64Var(&p.timeout, "timeout", 2, "timeout collection of messages after N seconds, default is 2 seconds")
```

Comment:
Unsure if it makes any sense to use milli/microseconds here, I specifically opted for int64 in case we want to change it to milliseconds/microseconds in the future. Very unlikely that even milliseconds would make any sense though.

Reply:
Thanks, will look into it!